data/hai/unfettered 0 prologue.txt

```diff
@@ -607,7 +608,8 @@ mission "Unfettered: Bring Hai"
 					goto brethren
 				`	"Why do you need me to do it?"`
 			label whyme
-			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high. The Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not many Hai captains are willing to come here. If they do, they charge large fees because of the obvious risks involved - for instance, getting killed in the crossfire. Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
+			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high, due to an increase in the amount of Hai wanting to come to our space. Even in normal times, the Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not that many Hai captains are willing to come here. When they do, some charge large fees because of the obvious risks involved - for instance, getting killed in the crossfire. Others are more generous, of course, but it is still not enough.`
+			`	Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
```

Comment:
```suggestion
			`	"Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
```
Not a huge fan of this tbh.

Reply:
wdym? you think he should say smth else or?

Reply:
The first sentence feels a bit too long and convoluted. It's not much of a problem since an alien says it, you can keep it if you want to.

Reply:
I think it's fine, I don't want him to have perfect english anyhow, in fact its good if he slips a bit
still, thx for the input!